examples/stable_diffusion_v2/text_to_image.py

```diff
@@ -156,15 +156,18 @@ def main(args):
         lora_only_ckpt=args.lora_ckpt_path,
     )
 
+    dpm_common_kwargs = {
+        "model_type": "v" if args.version == "2.1" else "noise",
```

Comment:
Not all sd 2.1 variants use v-prediciton. 
2.0-v: v_prediction. 
2.1-base: noise
2.1-v: v_prediction

We may follow the naming rule in hf. if it is not base 512 model, we add postfix like _768_v to version number, e.g. 2.0_768_v.  And judge prediction type by ending with "_v"